CMakeOpenCLPythonPackage.txt

```diff
@@ -0,0 +1,113 @@
+
+set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
+set(BOOST_VERSION_DOT "1.73")
+string(REPLACE "." "_" BOOST_VERSION_UNDERSCORE ${BOOST_VERSION_DOT})
+
+set(OPENCL_HEADER_REPOSITORY "https://github.com/KhronosGroup/OpenCL-Headers.git")
+set(OPENCL_HEADER_TAG "3f7c1f77f8e1a3c9b48765c713ca19013e95b91e")
+
+# FIXME -- remove reference to OpenCL ICD Loader fork after decision on whether or not to include Linux build
+#set(OPENCL_LOADER_REPOSITORY "https://github.com/KhronosGroup/OpenCL-ICD-Loader.git")
```

Comment:
can you please remove all commented-out code?

Reply:
Removed, see previous comment.